content/influxdb/cloud/v1-compatibilty-api.html

```diff
@@ -0,0 +1,565 @@
+---
+title: InfluxDB Cloud v1 compatiblity API documentation
```

Comment:
Is 1.x compatibility a subset of 2.0? Would it make sense to structure like this?:
/v2.0/api/
/v2.0/api/v1-compatibility